azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/service/calling/sdk/CallingSDKWrapper.kt

```diff
@@ -35,7 +35,7 @@ import java9.util.concurrent.CompletableFuture
 import kotlinx.coroutines.flow.Flow
 
 internal class CallingSDKWrapper(
```

Comment:
Update the wrapper to pull config via the config store and ID, instead of directly passing the field around.